src/main/java/dev/ebullient/convert/tools/pf2e/qute/QuteCreature.java

```diff
@@ -0,0 +1,52 @@
+package dev.ebullient.convert.tools.pf2e.qute;
+
+import java.util.Collection;
+import java.util.List;
+
+import dev.ebullient.convert.tools.Tags;
+import dev.ebullient.convert.tools.pf2e.Json2QuteCreature.CreatureLanguages;
+import dev.ebullient.convert.tools.pf2e.Pf2eSources;
+import io.quarkus.qute.TemplateData;
+
+/**
+ * Pf2eTools Creature attributes ({@code creature2md.txt})
+ * <p>
+ * Use `%%--` to mark the end of the preamble (frontmatter and
+ * other leading content only appropriate to the standalone case).
+ * </p>
+ * <p>
+ * Extension of {@link dev.ebullient.convert.tools.pf2e.qute.Pf2eQuteBase Pf2eQuteBase}
+ * </p>
+ */
+@TemplateData
+public class QuteCreature extends Pf2eQuteBase {
+
+    /** Aliases for this note (optional) */
+    public final List<String> aliases;
+    /** Collection of traits (decorated links, optional) */
+    public final Collection<String> traits;
+    /** Short creature description (optional) */
+    public final String description;
+    /** Creature level (number, optional) */
+    public final Integer level;
+    /** Creature perception (number, optional) */
+    public final Integer perception;
+    /** Languages as {@link CreatureLanguages} */
```

Comment:
Double-check the generated doc for this one. It may need to be the fully-qualified package name so the generated documentation can be traversed/cross-linked.

```suggestion
    /** Languages as {@link dev.ebullient.convert.tools.pf2e.Json2QuteCreature.CreatureLanguages} */
```